bin/start-shuffle-server.sh

```diff
@@ -31,6 +31,7 @@ JAR_DIR="${RSS_HOME}/jars"
 LOG_CONF_FILE="${RSS_CONF_DIR}/log4j2.xml"
 LOG_PATH="${RSS_LOG_DIR}/shuffle_server.log"
 AUDIT_LOG_PATH="${RSS_LOG_DIR}/audit.log"
+RPC_AUDIT_LOG_PATH="${RSS_LOG_DIR}/shuffle_server_audit.log"
```

Comment:
Perhaps we could choose a different name, as it might be confusing for people to see two logs both ending with the "audit" suffix.

If necessary, we could also rename `audit.log` to something else.